lib/portage/const.py

```diff
@@ -98,6 +98,11 @@
 LOCALE_DATA_PATH = f"{PORTAGE_BASE_PATH}/locale"  # FIXME: not used
 EBUILD_SH_BINARY = f"{PORTAGE_BIN_PATH}/ebuild.sh"
 MISC_SH_BINARY = f"{PORTAGE_BIN_PATH}/misc-functions.sh"
+JOBSERVER_BINARY = f"{PORTAGE_BIN_PATH}/jobserver"
```

Comment:
I get an error: "Could not find job server binary" when trying to install anything.

This is being installed as a system-wide binary, not an "internal Portage executable" (as the `meson_options.txt` describes the `portage-bindir` option, which gets propagated to `PORTAGE_BIN_PATH` via meson), so it should be `{BINARY_PREFIX}/usr/bin/jobserver` I think. At least it fixes the error for me.

Reply:
Yes, it should use the full the full path. fixed